sdk/storage/Azure.Storage.Blobs/src/Generated/BlobRestClient.cs

```diff
@@ -15655,15 +15655,15 @@ public static partial class BlobsModelFactory
         /// Creates a new AccountInfo instance for mocking.
         /// </summary>
         public static AccountInfo AccountInfo(
-            Azure.Storage.Blobs.Models.SkuName skuName,
+            bool isHierarchicalNamespaceEnabled,
             Azure.Storage.Blobs.Models.AccountKind accountKind,
-            bool isHierarchicalNamespaceEnabled)
+            Azure.Storage.Blobs.Models.SkuName skuName)
```

Comment:
Why did you change Storage's generated clients?
Btw. some of these changes might be breaking.
FYI @seanmcc-msft